<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>Dialogcoordtrans</class>
 <widget class="QDialog" name="Dialogcoordtrans">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>640</width>
    <height>277</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>Dialog</string>
  </property>
  <widget class="QDialogButtonBox" name="buttonBox">
   <property name="geometry">
    <rect>
     <x>540</x>
     <y>40</y>
     <width>81</width>
     <height>241</height>
    </rect>
   </property>
   <property name="orientation">
    <enum>Qt::Vertical</enum>
   </property>
   <property name="standardButtons">
    <set>QDialogButtonBox::Cancel|QDialogButtonBox::Ok</set>
   </property>
  </widget>
  <widget class="QLabel" name="label">
   <property name="geometry">
    <rect>
     <x>50</x>
     <y>90</y>
     <width>141</width>
     <height>17</height>
    </rect>
   </property>
   <property name="text">
    <string>输入文件路径</string>
   </property>
  </widget>
  <widget class="QLineEdit" name="lineEditSource">
   <property name="geometry">
    <rect>
     <x>40</x>
     <y>120</y>
     <width>391</width>
     <height>27</height>
    </rect>
   </property>
  </widget>
  <widget class="QLabel" name="label_2">
   <property name="geometry">
    <rect>
     <x>50</x>
     <y>160</y>
     <width>141</width>
     <height>17</height>
    </rect>
   </property>
   <property name="text">
    <string>输出文件路径</string>
   </property>
  </widget>
  <widget class="QLineEdit" name="lineEditDest">
   <property name="geometry">
    <rect>
     <x>40</x>
     <y>180</y>
     <width>391</width>
     <height>27</height>
    </rect>
   </property>
  </widget>
  <widget class="QPushButton" name="pushButtonSource">
   <property name="geometry">
    <rect>
     <x>480</x>
     <y>120</y>
     <width>41</width>
     <height>27</height>
    </rect>
   </property>
   <property name="text">
    <string>...</string>
   </property>
  </widget>
  <widget class="QPushButton" name="pushButtonDest">
   <property name="geometry">
    <rect>
     <x>480</x>
     <y>180</y>
     <width>41</width>
     <height>27</height>
    </rect>
   </property>
   <property name="text">
    <string>...</string>
   </property>
  </widget>
  <widget class="QRadioButton" name="radioButton_BasedOnLatLon">
   <property name="geometry">
    <rect>
     <x>50</x>
     <y>40</y>
     <width>116</width>
     <height>22</height>
    </rect>
   </property>
   <property name="text">
    <string>根据经纬度</string>
   </property>
   <property name="checked">
    <bool>true</bool>
   </property>
  </widget>
  <widget class="QRadioButton" name="radioButton_BasedOn3DCoord">
   <property name="geometry">
    <rect>
     <x>240</x>
     <y>40</y>
     <width>116</width>
     <height>22</height>
    </rect>
   </property>
   <property name="text">
    <string>根据三维坐标</string>
   </property>
  </widget>
 </widget>
 <resources/>
 <connections>
  <connection>
   <sender>buttonBox</sender>
   <signal>accepted()</signal>
   <receiver>Dialogcoordtrans</receiver>
   <slot>accept()</slot>
   <hints>
    <hint type="sourcelabel">
     <x>248</x>
     <y>254</y>
    </hint>
    <hint type="destinationlabel">
     <x>157</x>
     <y>274</y>
    </hint>
   </hints>
  </connection>
  <connection>
   <sender>buttonBox</sender>
   <signal>rejected()</signal>
   <receiver>Dialogcoordtrans</receiver>
   <slot>reject()</slot>
   <hints>
    <hint type="sourcelabel">
     <x>316</x>
     <y>260</y>
    </hint>
    <hint type="destinationlabel">
     <x>286</x>
     <y>274</y>
    </hint>
   </hints>
  </connection>
 </connections>
</ui>
